index.css

```diff
@@ -1,16 +1,44 @@
 body {
```

Comment:
Kindly note that for this project we actually want our design to as much as possible closely follow [the wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png). Kindly ensure to improve upon the following **styling** which are **missing** in your design:
- [ ] OPTIONAL
The _List page_ should have the table styled in **database alternate row**, as can be seen in the wireframe design
- [ ] OPTIONAL
To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md).
- [ ] OPTIONAL
Kindly note that the add button is **shorter** and it's **aligned to the right** with the form inputs. Kindly endeavor to improve upon this aspect.
- [ ] OPTIONAL
Kindly note that each page has a footer with copyright aligned to the left of the page

**Your designs:**
![image](https://user-images.githubusercontent.com/12745474/138296764-0d4a248e-2d9e-48ea-8c59-f8ab119e396a.png)

![image](https://user-images.githubusercontent.com/12745474/138296837-7c478bc3-762e-4c5c-a690-2d771625d327.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/138297081-3f10d3e9-43e6-44c4-b1de-be06ddb73e4b.png)